docs/i18n.md

```diff
@@ -20,96 +20,11 @@ NODE_PATH='./:./src' bin/create-locales
-
-The strings are extracted using a babel plugin via webpack. Extracted strings are added to a pot template file. This file is used to seed the po for each locale with the strings needing translating when merging locales.
-
-Run `git diff` to see what the extraction did. **If no strings were updated then you do not have to continue creating the pull request. You can revert the changes made to the `pot` timestamp.** Here is an example of a diff where no strings were changed. It just shows a single change to the timestamp:
-
-```diff
-diff --git a/locale/templates/LC_MESSAGES/amo.pot b/locale/templates/LC_MESSAGES/amo.pot
-index 31e113f2..c7da4e34 100644
---- a/locale/templates/LC_MESSAGES/amo.pot
-+++ b/locale/templates/LC_MESSAGES/amo.pot
-@@ -2,7 +2,7 @@ msgid ""
- msgstr ""
- "Project-Id-Version: amo\n"
- "Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
--"POT-Creation-Date: 2017-06-08 14:01+0000\n"
-+"POT-Creation-Date: 2017-06-08 14:43+0000\n"
- "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
- "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
- "Language-Team: LANGUAGE <LL@li.org>\n"
-```
-
-When the application is under active development it's more likely that you will see a diff containing new strings or at least strings that have shifted to different line numbers in the source. If so, commit your change and continue to the next step:
-
-```
-git commit -a -m "Extract AMO locales"
-```
-
-#### Merging locale files
-
-After extracting new strings, you have to merge them into the existing locale files. Do this in your branch and commit:
-
-```
-bin/merge-locales
-```
-
-Keep an eye out for [fuzzy strings](https://www.gnu.org/software/gettext/manual/html_node/Fuzzy-Entries.html) by running `git diff` and searching for a comment that looks like `# fuzzy`. This comment means the localization may not exactly match the source text; a localizer needs to review it. As per our configuration, the application will not display fuzzy translations. These strings will fall back to English.
-
-In some rare cases you may wish to remove the `fuzzy` marker to prevent falling back to English. Discuss it with a team member before removing `fuzzy` markers.
-
-Commit and continue to the next step:
-
-```
-git commit -a -m "Merged AMO locales"
-```
-
-#### Finalizing the extract/merge process
-
-Now that you have extracted and merged locales for one application, it's time to create a pull request for your branch. For example:
-
-```
-git push origin amo-locales
-```
-
-If the pull request passes all of our CI tests it is likely good to merge. You don't need to ask for a review unless you're unsure of something because often locale updates will be thousands of lines of minor diffs that can't be reasonably reviewed by a human. 🙂 If the pull request passes all of our CI tests it is likely good to merge.
-
-#### Building the JS locale files
```

Comment:
It's not totally clear to me what this is for and how it get's run in our local development flow and or in CI. Do I need to integrate this logic to build the json files as a part of `extract-locales` should be simple enough but I'm not sure if this is needed.